core/document/src/document.rs

```diff
@@ -63,6 +63,12 @@ impl Document {
 		svg
 	}
 
+	// Returns string that represents the full document (not just svg)
+	pub fn serialize_document(&mut self) -> String {
+		// Manually constructing JSON. Should switch to serde_json
+		format!("{{\"version\": 0, \"svg\": \"{}\"}}", self.render_root())
```

Comment:
It would also be good to get feedback on what we think the serialized document should look like. Here I am constructing a JSON string with the `version` and `svg` keys. If we are okay with this, I can switch to constructing the JSON properly.

Reply:
Shouldn"t the forma be something we can import again?